src/Controller/QueryFieldController.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\EzPlatformQueryFieldType\Controller;
+
+use EzSystems\EzPlatformQueryFieldType\API\QueryFieldService;
+use eZ\Publish\Core\MVC\Symfony\View\ContentView;
+
+class QueryFieldController
```

Comment:
Is it possible to make it `final`? _please also apply this comment to other controllers_